indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/X509SignatureAlgorithm.kt

```diff
@@ -51,13 +51,13 @@ enum class X509SignatureAlgorithm(
         return Asn1.Sequence {
```

Comment:
Maybe refactor to expression body (after making `shaOid` an extension function)

Reply:
opted against an extension, as it is very specific, but I did refactor it to an expression body